examples/02_decoding/plot_haxby_different_estimators.py

```diff
@@ -71,14 +71,18 @@
                       param_grid={'C': [.1, .5, 1., 5., 10., 50., 100.]},
                       scoring='f1', n_jobs=1, cv=3, iid=False)
 
-logistic_cv = GridSearchCV(LogisticRegression(C=1., penalty="l1"),
-                           param_grid={'C': [.1, .5, 1., 5., 10., 50., 100.]},
-                           scoring='f1', cv=3, iid=False)
-logistic_l2_cv = GridSearchCV(LogisticRegression(C=1., penalty="l2"),
-                              param_grid={
-                                  'C': [.1, .5, 1., 5., 10., 50., 100.]
-                              },
-                              scoring='f1', cv=3, iid=False)
+logistic_cv = GridSearchCV(
```

Comment:
Did you change any parameters to make it more interesting. What are you trying to fix ?

Reply:
The default solver for LogisticsRegression is going to be set to a different default in future and in the examples that have not specified it explicitly, a FutureWarning is being generated, making for a super long log with pointless info.
I am setting those examples to explicitly use the original (soon to be former) default, so the FutureWarning goes away.